sdk/pixKey/pixKey.js

```diff
@@ -0,0 +1,226 @@
+};
+
+exports.query = async function ({ limit, after, before, status, tags, ids, type, user } = {}) {
+    /**
+     *
+     * Retrieve PixKeys
+     *
+     * @description Receive a generator of PixKeys objects previously created in the Stark Infra API
+     *
+     * Parameters (optional):
+     * @param limit [integer, default 100]: maximum number of objects to be retrieved. Max = 100. ex: 35
+     * @param after [string, default null]: date filter for objects created after a specified date. ex: '2020-03-10'
+     * @param before [string, default null]: date filter for objects created before a specified date. ex: '2020-03-10'
+     * @param status [list of strings, default null]: filter for status of retrieved objects. Options: "created", "failed", "delivered", "closed", "canceled".
+     * @param ids [list of strings, default null]: list of ids to filter retrieved objects. ex: ["5656565656565656", "4545454545454545"]
+     * @param type [list of strings, default null]: filter for the type of retrieved PixKeys. Options: "fraud", "reversal", "reversalChargeback"
+     * @param user [Organization/Project object, default null]: Organization or Project object. Not necessary if starkinfra.user was set before function call
+     *
+     * Return:
+     * @returns generator of PixKey objects with updated attributes
+     *
+     */
+    let query = {
+        limit: limit,
+        after: check.date(after),
+        before: check.date(before),
+        status: status,
+        tags: tags,
+        ids: ids,
+        type: type,
+    };
+    return rest.getList(resource, query, user);
+};
+
+exports.page = async function ({ cursor, limit, after, before, status, tags, ids, type, user } = {}) {
+    /**
+     *
+     * Retrieve paged PixKeys
+     *
+     * @description Receive a list of up to 100 PixKey objects previously created in the Stark Infra API and the cursor to the next page.
+     * Use this function instead of query if you want to manually page your requests.
+     *
+     * Parameters (optional):
+     * @param cursor [string, default null]: cursor returned on the previous page function call
+     * @param limit [integer, default 100]: maximum number of objects to be retrieved. Max = 100. ex: 35
+     * @param after [string, default null]: date filter for objects created after a specified date. ex: '2020-03-10'
+     * @param before [string, default null]: date filter for objects created before a specified date. ex: '2020-03-10'
+     * @param status [list of strings, default null]: filter for status of retrieved objects. Options: "created", "failed", "delivered", "closed", "canceled".
+     * @param ids [list of strings, default null]: list of ids to filter retrieved objects. ex: ["5656565656565656", "4545454545454545"]
+     * @param type [list of strings, default null]: filter for the type of retrieved PixKeys. Options: "fraud", "reversal", "reversalChargeback"
```

Comment:
type não é uma lista de strings. É um string dentre as opções: "cpf", "cnpj", "phone", "email" and "evp"